portal-web/test/functional/com/liferay/portalweb/paths/portlet/navigation/Navigation.path

```diff
@@ -39,6 +39,24 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>SCREENSHOTS_SIGN_IN</td>
+	<td>portal/screenshots/user/sign_in.png</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>SCREENSHOTS_LOG_IN</td>
+	<td>portal/screenshots/user/log_in.png</td>
+	<td></td>
+</tr>
```

Comment:
Let's have the name parallel existing paths and have it sit in the respective path file.
ie: UserBar#USER_SIGN_IN_PNG

My bad, for not catching this on nick's commit.

Reply:
It doesn't look like we have a standard in our path files overall...some path names do not have PNG or indication that it's sikuli path other than the locator points to the sikuli directory. But OS.path is a good example that I'd want us to follow.